### DISCUSSION TITLE
Scrum Prep Milestone #2 - Fourth Week

### DISCUSSION THREAD

**Discussion by YanehRuiz** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#291] [Weekly Retrospective]
Weekly Development Update: Fourth week of Milestone 2

Hello team!

Please provide a brief comment detailing what you accomplished this week, including any challenges faced or insights gained. Make sure to submit your updates by the deadline, which is before: Friday, October 18, 2024 @ 11am.

Thank you for your collaboration!

As for my collaboration:
I've graded, approved and reviewed issues and attended weekly meetings.

**Comment by evnd04** [comment]
Same as @YanehRuiz ive mostly been reviewing issues and approving/grading them. Also leaving feedback on issues and how to improve redaction.

**Comment by Jjulianirizarry24** [comment]
Created new issues related to testing. Attended meetings. Got started working on corresponding issues, will be done by next week.

**Comment by javier-rivera8** [comment]
This week, I didn’t have much time because of tests. I will start as soon as possible and aim to finish my tasks by next week.

**Comment by johnsantiago9** [comment]
This week i wasn’t able to work due to exams. Will be touching base with my tasks during the weekend

**Comment by JuanJRivera** [comment]
I won't be able to work this week because of exams. This weekend I am gonna continue with my tasks.

**Comment by Alejandro-DeJesus** [comment]
Directed the meeting for my team, and created issues for the final strech of the milestone. But overall it was a slow week as I directed my efforts towards other classes.

**Comment by ghost** [comment]
Worked on improving dummy data, will try to finish more tasks this weekend.

**Comment by jonVargas1** [comment]
Started working on issue #275 but didn't get much done due to work for other classes

**Comment by yadrielsamir** [comment]
Created new issues to work on the week and attended meetings. Worked on current issues, should be done by this weekend.

**Comment by RaquelQuinones** [comment]
This week I was not able to do anything due to getting sick.

**Comment by joel-alvarado** [comment]
Research LTT creation and looked into writing a security report on the app.

**Comment by misaelmercado1** [comment]
Started working on button component for the list of reports but implementation proved a little bit more difficult than intended so I will work with that over the weekend

**Comment by lex939** [comment]
I've helped in closing several documentation issues as we head into the final week of the milestone.

In this final week, I hope to get the documentation (with visible changes and a legend) ready for submission.

**Comment by JorgeC08** [comment]
Researched the Lecture Topic Task (LTT) to start working once I finish another issue that I ave with another class.

**Comment by NathBat09** [comment]
Research, and completing my tasks

**Comment by Chosen-Juan1** [comment]
Finished the admin communications interface page on figma.

**Comment by CaribbeanCool** [comment]
Worked on my 2 issues and opened the PR #289

**Comment by Rodal1** [comment]
Haven't been able to do much work, this week, I've been very busy out of campus. All I've been able to work is Lecture Topic Task.

**Comment by noel-vargas** [comment]
I didn't really work on the project this week. I was busy with other classes

**Comment by JorgeRivera94** [comment]
Posted exploration and results for updated report labels. Plans for the weekend and next week include identifying development and Lecture Topic issues.
